hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/contracts/AbstractLedgerEvmWorldUpdater.java

```diff
@@ -38,15 +39,21 @@
     private Map<Address, UpdatedHederaEvmAccount<A>> updatedEvmAccounts = new HashMap<>();
     private HederaEvmEntityAccess hederaEvmEntityAccess;
 
+    private TokenAccessor tokenAccessor;
+
     protected AbstractLedgerEvmWorldUpdater(final W world, final AccountAccessor accountAccessor) {
         this.world = world;
         this.accountAccessor = accountAccessor;
     }
 
     protected AbstractLedgerEvmWorldUpdater(
-            final W world, final AccountAccessor accountAccessor, final HederaEvmEntityAccess hederaEvmEntityAccess) {
+            final W world,
+            final AccountAccessor accountAccessor,
+            final HederaEvmEntityAccess hederaEvmEntityAccess,
+            TokenAccessor tokenAccessor) {
```

Comment:
We can make `TokenAccessor` final and for better code organisation, we can pass `tokenAccessor` right after `accountAccessor`.